skimage/filters/__init__.py

```diff
@@ -1,3 +1,19 @@
+"""Filters subpackage.
+
+Filters perform mathematical operations to reduce or enhance certain aspects
+of a 2D or higher dimensional image [1]_. The subpackage contains, e.g.,
+filters to blur the original image and to detect blob- or edge-like structures.
+
+.. [1] https://en.wikipedia.org/wiki/Digital_image_processing#Filtering
+
+Subpackages
```

Comment:
This part will not be rendered well in the API page of the documentation (http://scikit-image.org/docs/stable/api/api.html)

Reply:
@emmanuelle can you elaborate on why?